src/linear_algebra/unitary_group.lean

```diff
@@ -64,11 +64,19 @@ def unitary_group : Type* := unitary_submonoid n α
 end
 
 variables {α : Type v} [comm_ring α] [star_ring α]
+variables {n : Type u} [decidable_eq n] [fintype n]
 
-namespace unitary_group
+namespace unitary_submonoid
 
-variables {n : Type u} [decidable_eq n] [fintype n]
+lemma star_mem {A : matrix n n α} (hA : A ∈ unitary_submonoid n α) :
```

Comment:
If you make an iff version of this using star_star then it can be a simp lemma (like neg_mem_iff)